src/js/components/organisms/brands2.scss

```diff
@@ -39,6 +38,9 @@
     }
 
     .footer-block {
-        margin-top: $buildingUnit * 5;
```

Comment:
if possible, move this scss file into the component in the nuxt project.

Reply:
Seeing as this is just a little hotfix/bugfix to fix the margins, wouldn't it be a little overhead to start moving files around? I mean, I'd rather just do this little fix now, and move the components when we're reworking the CSS